test/test-rotatesTetrominos.js

```diff
@@ -0,0 +1,32 @@
+var assert = require('chai').assert;
+var tetris = require('../tetris');
+describe("rotate L-tetromino", function (){
+    class Tetramino = {
+        constructor(tetramino, phase, coordinates){
+            this.typeTetramoni = tetramino
+            this.initialRotationPhase = phase;
+            this.initialCoordinates = coordinates;
```

Comment:
Parameters are "initial", not properties. Also for other properties

Reply:
i do not understand, what is wrong here? :(

Reply:
You have field named initialCoordinates , should be "coordinates" .
You have constructor parameter named "coordinates", should be initialCoordinates
And for other fields/parameters